.gitignore

```diff
@@ -6,4 +6,5 @@ __pycache__/
 **/*.swp
 data/
 nifti/
-.vscode/
\ No newline at end of file
+.vscode/
+*.h5
```

Comment:
To dodałam, bo te wagi będą się zmieniać co trening, a jak będziemy mieć gotowy ładny wytrenowany model to w sumie pasowałoby go jakoś oznaczyc, żeby wiedzieć, że to są końcowe wagi.

Reply:
Może odrazu cały folder z wagami?
A jak będziemy mieli wytrenowany, to po prostu puścim do innego folderu